modules/ROOT/pages/salesforce/salesforce-connector-design-center.adoc

```diff
@@ -0,0 +1,130 @@
+* *Subscribe Channel* - Subscribe to a streaming channel. This trigger provides channel-related notification settings for new events that take place after you subscribe.
+* *Subscribe Topic* - Subscribe to a topic. This trigger provides topic-related notification settings for new events that take place after you subscribe.
+
+== New Object Trigger
+
+The following shows how to configure the Salesforce New Object trigger.
+
+. Create a New Mule Application in Design Center.
+. Set the trigger. For example, you can use Salesforce > New Object.
+. Click *Set up*.
+. Specify the Connection Type:
++
+[%header%autowidth.spread]
+|===
+|Connection Type |Salesforce Information Site
+| <<Basic Username Password Authorization, Basic Username Password>> |https://developer.salesforce.com/docs/atlas.en-us.api.meta/api/sforce_api_calls_login.htm[SOAP API Username Password Login]
+| <<OAuth 2.0 Authorization, OAuth 2.0>> |https://help.salesforce.com/articleView?id=remoteaccess_oauth_web_server_flow.htm[OAuth 2.0 Web Server Authorization Flow]
+| <<OAuth JWT Authorization, OAuth JWT>> |https://help.salesforce.com/articleView?id=remoteaccess_oauth_jwt_flow.htm[OAuth 2.0 JWT Bearer Token Flow]
+| <<OAuth SAML Authorization, OAuth SAML>> |https://help.salesforce.com/apex/HTViewHelpDoc?id=remoteaccess_oauth_SAML_bearer_flow.htm[OAuth 2.0 SAML Bearer Assertion Flow]
+| <<OAuth Username Password Authorization, OAuth Username Password>> |https://help.salesforce.com/articleView?id=remoteaccess_oauth_username_password_flow.htm[OAuth 2.0 Username-Password Flow]
+|===
++
+NOTE: In version 9.6.0, OAuth 2.0 is listed as Deprecated. This connection type is not deprecated and you can ignore the `Deprecated` label. This was fixed in version 9.6.1 with the removal of the Deprecated label - the functionality didn't change, only its label.
++
+. Specify the parameters as required for each configuration type, and click *Test* to ensure you have a valid connection to Salesforce.
+. If needed, configure the *Apex* tab to decide what metadata to fetch and to add Apex class names.
+. If you haven't specified a `Connection Type`, and if needed, configure the *Advanced* tab to set the `Max Idle Time` and `Time Units` for how long to hold open a connection before it expires.
+. After you specify a connection type, if needed, you can use the *Advanced* tab to change values for the Reconnection Strategy, Pooling Profile, and Expiration Policy.
+. If needed, set the `Redelivery Policy` on the main configuration screen. This indicates the maximum number of tries to deliver trigger content.
+
+== Connector as a Component
+
+After creating a trigger, click the plus sign. If needed, click *Transform* to use DataWeave to map fields between your received data and what to use with your Salesforce component.
```

Comment:
Are we really going to refer to this connector as a component from now on? I find that confusing. If this is a thing we're going to do, it would be helpful to make it explicit here?

Reply:
In Design Center, the card is called Component. I'm just following the UI.